data/mods/Backrooms/mod_tiles.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "type": "mod_tileset",
+    "compatibility": [ "ChestHole32", "MShock32Modded", "MshockXottoplus", "MSX++DEAD_PEOPLE", "UNDEAD_PEOPLE", "UNDEAD_PEOPLE_BASE" ],
```

Comment:
this and the png should not get in this PR. Maybe submit it to the tileset repo if you want it added but not directly to the game.